GameData/ROCapsules/PartConfigs/GeminiBDB/RP_GeminiBDB.cfg

```diff
@@ -0,0 +1,53 @@
+//Star 13E retro
+@PART[ROC-GeminiRetroBDB]:FOR[RealPlume]:NEEDS[SmokeScreen]
+{
+	PLUME
+	{
+		name = Solid-Sepmotor
+		transformName = thrustTransform
+		plumePosition = 0.0, 0.0, 0.0
+		plumeScale = 0.25
+		flarePosition = 0.0, 0.0, 0.0
+		flareScale = 1.0
+		smokePosition = 0.0, 0.0, 0.0
+		smokeScale = 1.0
+		slagPosition = 0.0, 0.0, 0.0
+		slagScale = 1.0
+		localRotation = 0.0, 0.0, 0.0
+		emissionMult = 0.5
+		energy = 1.0
+		speed = 1.25
+	}
+
+	@MODULE[ModuleEnginesRF]:HAS[#thrustVectorTransformName[thrustTransform]]
+	{
+		%powerEffectName = Solid-Sepmotor
+	}
+}
+
+//Gemini B sepmotors
+@PART[ROC-GeminiBEquipmentSection]:FOR[RealPlume]:NEEDS[SmokeScreen]
```

Comment:
Do not declare a :FOR pass that is not for our own mod. Must use BEFORE or AFTER